src/main/include/AlphaArm.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2023-2024 CurtinFRC
+// Open Source Software, you can modify it according to the terms
+// of the MIT License at the root of this project
+
+#pragma once
+#include <frc/DigitalInput.h>
+
+#include "Wombat.h"
+
+struct AlphaArmConfig {
+  // wom::Gearbox armGearBox;
+  wom::Gearbox alphaArmGearbox;
+  wom::Gearbox wristGearbox;
+};
+
+enum class AlphaArmState { kIdle, kIntakeAngle, kAmpAngle, kSpeakerAngle, kRaw };
+
+class AlphaArm : public ::behaviour::HasBehaviour {
+ public:
+  explicit AlphaArm(AlphaArmConfig config);
+
+  void OnUpdate(units::second_t dt);
+  void SetRaw(units::volt_t volt);
```

Comment:
Rename to SetVolts or SetRawVolts
Should probably take in a voltage for each Voltage variable